src/Progress/CircularProgress.js

```diff
@@ -53,18 +67,37 @@ export const styleSheet = createStyleSheet('MuiCircularProgress', theme => ({
 }));
 
 function CircularProgress(props) {
-  const { classes, className, size, ...other } = props;
+  const { classes, className, size, mode, value, min, max, ...other } = props;
   const radius = size / 2;
+  const rootProps = {};
+  const svgClasses = classNames(classes.svg, {
+    [classes.indeterminateSvg]: mode === 'indeterminate',
+  });
+
+  const circleClasses = classNames(classes.circle, {
+    [classes.indeterminateCircle]: mode === 'indeterminate',
+    [classes.determinateCircle]: mode === 'determinate',
+  });
+
+  const styles = { circle: {} };
+  if (mode === 'determinate') {
+    const relVal = getRelativeValue(Math.round(value), min, max);
+    styles.circle.strokeDasharray = `calc(((100% - ${THICKNESS}px) * ${PI}) * ${relVal}),` +
+                                    `calc((100% - ${THICKNESS}px) * ${PI})`;
+    rootProps['aria-valuenow'] = Math.round(value);
```

Comment:
Should we add the `role="progressbar"` and `aria-valuemin="0" aria-valuemax="100"` when we have the information?
Polymer is a good reference:  https://www.webcomponents.org/element/PolymerElements/paper-progress